examples/bevy/README.md

```diff
@@ -24,14 +33,15 @@ To link gameplay to a particular player, you may want to pair this demo with the
 [ECDSA]: https://github.com/risc0/risc0/tree/main/examples/ecdsa
 
 ## Project Organization
+
 zkVM applications are organized into a [host program] and a [guest program].
 The host program can be found in [`src/main.rs`] and the guest program can be found in [`methods/guest/src/main.rs`].
 
 The [host] first [executes] the guest program and then [proves the execution] to construct a [receipt].
 The receipt can be passed to a third party, who can examine the [journal] to check the program's outputs and can [verify] the [receipt] to ensure the integrity of the [guest program]'s execution.
 
-[`src/main.rs`]: https://github.com/risc0/risc0/tree/v0.16.1/examples/bevy/src/main.rs
-[`methods/guest/src/main.rs`]: https://github.com/risc0/risc0/tree/v0.16.1/examples/bevy/methods/guest/src/main.rs
+[`src/main.rs`]: src/main.rs
```

Comment:
@tzerrell and I have recently been expressing some uncertainty about whether the relative paths or full url paths are preferable in this context